buildtools/cli.go

```diff
@@ -325,9 +335,13 @@ func GetCommands() []cli.Command {
 			SkipFlagParsing: true,
 			BashComplete:    corecommon.CreateBashCompletionFunc("install", "i", "isntall", "add", "ci", "publish", "p"),
 			Category:        buildToolsCategory,
-			Action: func(c *cli.Context) error {
+			Action: func(c *cli.Context) (errFromCmd error) {
 				cmdName, _ := getCommandName(c.Args())
-				return npmGenericCmd(c, cmdName, false)
+				if errFromCmd = npmGenericCmd(c, cmdName, false); errFromCmd != nil {
```

Comment:
This if statement that checks whether the error is nil or not should be placed inside the jfrog-cli-security module. This is to reduce the dependecy of jfrog-cli-security on this module, in case changes are needed to the curation logic.
The approach taken for "pip" is better, so let's adopt it here as well - 
<img width="913" alt="image" src="https://github.com/user-attachments/assets/ed2ab974-06be-4250-9ef6-4e5bd72994a7">

Reply:
its a bit more complicated, I will check again